### PR TITLE
Add popout window link

### PR DESCRIPTION
**Pull request by kelseygonzalez**

### CODE REVIEW

**Comment by kelseygonzalez** [comment]
@alex-r-bigelow @zoidy

File: resbaz/resbazTucson2021.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ May 17&ndash;21, 2021
 
 ## Schedule
 
-Note: Due to the "unconference" nature of ResBaz, you can opt to teach a workshop during the [registration](#register) process. Therefore, the schedule will not be final until the day of! Below represents the current schedule.
+Note: Due to the "unconference" nature of ResBaz, you can opt to teach a workshop during the [registration](#register) process. Therefore, the schedule will not be final until the day of! Below represents the current schedule. Click [here](https://kelseygonzalez.github.io/resbaz2021schedule/) to view the schedule in a new window.
 
 <div style="width:100%;overflow-x:auto">
 <iframe src="https://kelseygonzalez.github.io/resbaz2021schedule/" width="830" height="650" scrolling="auto" frameborder="no"></iframe>
```
Comment (alex-r-bigelow) [inline resbaz/resbazTucson2021.md:23]:
```suggestion
<iframe src="https://kelseygonzalez.github.io/resbaz2021schedule/" width="830" height="750" scrolling="auto" frameborder="no"></iframe>
```

Sorry, meant to get to this over the weekend. This should add a little more vertical space so less is clipped naturally.

**Review by alex-r-bigelow — APPROVED** [review]

**Comment by alex-r-bigelow** [comment]
@julianpistorius, were you the one with the privileges to give @kelseygonzalez push access?

**Comment by kelseygonzalez** [comment]
@julianpistorius should be, but I'm not sure @alex-r-bigelow

**Comment by alex-r-bigelow** [comment]
Sorry should have just merged it.

**Pull request merged**